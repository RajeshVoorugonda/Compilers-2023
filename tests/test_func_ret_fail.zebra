
@ Testing invalid returns
func int test(){
    return false;
}