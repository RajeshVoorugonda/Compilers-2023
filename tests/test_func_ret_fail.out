[1;31mtypeCheckError[0m([96mline:[32m2[0m): Not matching the expected return type of the function
